ote/src/cljs/ote/ui/table.cljs

```diff
@@ -78,9 +79,9 @@
         (doall
           (map-indexed
             (fn [i row]
-              ^{:key (if key-fn
-                       (key-fn row)
-                       (str i "-" random-table-id "-body-row"))}
+              ^{:key (str table-id "-b-row-" (if (and key-fn (key-fn row))
```

Comment:
This `key-fn` handling could have been removed, based on testing views, but to avoid any risk of regression I left it still there. Just in case with some data it's better.